src/Sulu/Bundle/AdminBundle/Resources/js/components/MultiItemSelection/item.scss

```diff
@@ -28,6 +43,10 @@ $itemBackgroundColor: $white;
     padding: 13px 10px 0 0;
     cursor: pointer;
     background-color: transparent;
+
+    &.hidden {
+        display: none;
```

Comment:
Why don't we simply not render the components instead?

Reply:
not rendering the elements is okay for me too 🙂